### PR TITLE
update python config guide

### PR DESCRIPTION
**Pull request by changlong-liu**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: documentation/code-gen/configure-python-sdk.md [hunk 1]
```diff
@@ -142,10 +142,7 @@ python:
 no-namespace-folders: true
 output-folder: $(python-sdks-folder)/appconfiguration/azure-mgmt-appconfiguration/azure/mgmt/appconfiguration
 ```
-``` yaml $(python-mode) == 'create'
-python:
-  basic-setup-py: true
-  output-folder: $(python-sdks-folder)/appconfiguration/azure-mgmt-appconfiguration
+``` yaml $(python-mode) == 'create' && $(track2)
```
Comment (qiaozha) [inline documentation/code-gen/configure-python-sdk.md:145]: How about this? https://github.com/Azure/azure-rest-api-specs/pull/11542/files#diff-694a0a1176057a399f20a5338d48566e5501d2c8d34e57f82a6d1272c28114d3L135-L144

**Comment by akning-ms** [comment]
HI @msyyc,  can you sign off this PR?
Hi @changlong-liu , you may need to reply @qiaozha 's comments

**Comment by changlong-liu** [comment]
yes, updated in latest commit. @qiaozha , @msyyc, @akning-ms

File: documentation/code-gen/configure-python-sdk.md (hunk 1 above)
Comment (changlong-liu) [reply documentation/code-gen/configure-python-sdk.md:152]: yes, committed

**Review by qiaozha — APPROVED** [review]

**Comment by qiaozha** [comment]
LGTM

**Review by jsntcy — APPROVED** [review]

**Comment by akning-ms** [comment]
looks there some conflict, @changlong-liu can you resolve it first?

**Comment by changlong-liu** [comment]
Hi @akning-ms , the conflict resolved. Shall we merge this PR? Thanks.

**Pull request merged**